extensions/mssql/package.json

```diff
@@ -97,7 +97,7 @@
         },
         "mssql.tracingLevel": {
           "type": "string",
-          "description": "[Optional] Log level for backend services. This reuses existing files for logging by appending across restart. The default tracingLevel does not log much. User is responsible for cleaning out the old logs if this setting is changed to increase log verbosity. Error includes Critical, Warning includes Error, Information includes Warning and Verbose includes Information",
+          "description": "[Optional] Log level for backend services. Azuredatastudio generates a file name every time it starts and if the file already exists the logs entries are appended to that file. For cleanup of old log files see logRetentionMinutes and logFilesRemovalLimit settings. The default tracingLevel does not log much. User is responsible for cleaning out the old logs if this setting is changed to increase log verbosity. Error includes Critical, Warning includes Error, Information includes Warning and Verbose includes Information",
```

Comment:
"Azuredatastudio" should be "Azure Data Studio"

Reply:
fixed. Thanks!